src/no/runsafe/clans/commands/CreateClan.java

```diff
@@ -20,7 +20,7 @@ public CreateClan(IScheduler scheduler, ClanHandler clanHandler, CharterHandler
 	@Override
 	public String OnAsyncExecute(IPlayer executor, IArgumentList parameters)
 	{
-		String clanName = parameters.get("clanTag").toUpperCase();
+		String clanName = ((String) parameters.getValue("clanTag")).toUpperCase();
```

Comment:
Maybe add an .upperCase() to the argument specification so this can be made cleaner?